tools/PI/FeatureExploration/WindowPI-FeatureExploration.md

```diff
@@ -0,0 +1,140 @@
+# Dev Home Feature Exploration: Dev Home PI
```

Comment:
Feature Exploration for PI is already posted. Do we need this folder?

Reply:
I don't think so.